invoice/src/main/resources/org/killbill/billing/invoice/dao/InvoiceItemSqlDao.sql.stg

```diff
@@ -113,3 +113,22 @@ and <accountRecordIdField("ii.")> = :accountRecordId
 <AND_CHECK_TENANT("ii.")>
 ;
 >>
+
+getRepairMap(ids) ::= <<
```

Comment:
Can you check the `EXPLAIN` to make sure the cross-join doesn't explode on our large deployments? 😅

Reply:
Sure thing - looks pretty good:

```
+----+-------------+-------+------------+--------+---------------------------------------------------------------------------------------------------------------+----------------------------------------+---------+-------------------------------------+------+----------+------------------------------+
| id | select_type | table | partitions | type   | possible_keys                                                                                                 | key                                    | key_len | ref                                 | rows | filtered | Extra                        |
+----+-------------+-------+------------+--------+---------------------------------------------------------------------------------------------------------------+----------------------------------------+---------+-------------------------------------+------+----------+------------------------------+
|  1 | SIMPLE      | ii2   | NULL       | ref    | invoice_items_id,invoice_items_invoice_id,invoice_items_linked_item_id,invoice_items_tenant_account_record_id | invoice_items_tenant_account_record_id | 8       | const                               |    6 |    16.67 | Using where; Using temporary |
|  1 | SIMPLE      | i     | NULL       | eq_ref | invoices_id,invoices_tenant_account_record_id                                                                 | invoices_id                            | 110     | databasee9f4ebc7.ii2.invoice_id     |    1 |    33.33 | Using where                  |
|  1 | SIMPLE      | ii1   | NULL       | eq_ref | invoice_items_id,invoice_items_invoice_id,invoice_items_tenant_account_record_id                              | invoice_items_id                       | 110     | databasee9f4ebc7.ii2.linked_item_id |    1 |    16.67 | Using where                  |
+----+-------------+-------+------------+--------+---------------------------------------------------------------------------------------------------------------+----------------------------------------+---------+-------------------------------------+------+----------+------------------------------+

```